charts/fybrik/templates/fybrik-adminCRs-role.yaml

```diff
@@ -19,6 +19,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - services
```

Comment:
Why would we need to access services in the namespace where fybrikmodules and fybrikstorageaccounts are deployed?

Reply:
the vault service ?

Reply:
Vault service is deployed in fybrik namespace, e.g., fybrik-system. These could be different namespaces. The right place is fybrik-ctrl-role.yaml

Reply:
this change is mistake, we don't need it at all, there is a separate role for NP- fybrik-np-services-rb.yaml.